common/achievement/util.ts

```diff
@@ -116,6 +115,18 @@ export async function getBucketContext(userID: string, relation?: string): Promi
     return achievementContext;
 }
 
+export function filterBucketEvents(bucketEvents: BucketEvents[]) {
```

Comment:
Can you please add a comment to the function to explain why we are doing this?

Reply:
DONE!